src/main/resources/crafter/studio/upgrade/pipelines.yaml

```diff
@@ -721,6 +721,20 @@ pipelines:
           commitDetails: Add default gitignore file
         - type: versionFileUpgrader
 
+    - currentVersion: 3.1.4.2
+      nextVersion: 3.1.4.3
```

Comment:
This version number really looks out of date. I see in git log as latest 3.1.4.9

Reply:
This is for the `site` pipeline, not `system` https://github.com/craftercms/studio/blob/develop/src/main/resources/crafter/studio/upgrade/pipelines.yaml#L716

Reply:
Yes, I noticed. When I was looking for DB version update noticed it is in site pipeline